scripts/calculate_regressions.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import dataclasses
+import json
+from dataclasses import dataclass
+from typing import List, Iterable, Optional
+
+
+@dataclass(frozen=True)
+class Annotation(object):
+    filePath: str
+    line: int
+    column: int
+    highlightedText: str
+    error: str
+    inspectionToolId: Optional[str]
+
+    def __str__(self) -> str:
+        if self.inspectionToolId is not None:
+            suffix = f" by {self.inspectionToolId}"
+        else:
+            suffix = ""
+        return f"{self.filePath}:{self.line}:{self.column} '{self.highlightedText}' ({self.error}){suffix}"
+
+    @staticmethod
+    def from_dict(raw_dict):
```

Comment:
`def from_dict(raw_dict: Dict[str]) -> "Annotation":`?

Reply:
🤦 
It's very consistent syntax 😃